code/game/objects/items/twohanded.dm

```diff
@@ -1344,3 +1344,82 @@
 	hitsound = wielded ? 'yogstation/sound/weapons/bat_hit.ogg' : 'sound/items/trayhit1.ogg' //big donk if wielded
 	item_state = "bigspoon[wielded]" //i don't know why it's item_state rather than icon_state like every other wielded weapon
 	return
+
+
+/*
+Broom
+*/
+
+#define BROOM_PUSH_LIMIT 20
+/obj/item/twohanded/broom
+	name = "broom"
+	desc = "This is my BROOMSTICK! It can be used manually or braced with two hands to sweep items as you move. It has a telescopic handle for compact storage."
+	icon = 'icons/obj/janitor.dmi'
+	icon_state = "broom0"
+	lefthand_file = 'icons/mob/inhands/equipment/custodial_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/custodial_righthand.dmi'
+	force = 8
+	throwforce = 10
+	throw_speed = 3
+	throw_range = 7
+	w_class = WEIGHT_CLASS_NORMAL
+	force = 8
+	force_wielded = 12
```

Comment:
you have force twice, 
also, these stats are kinda ridiculous
force_wielded is the bonus damage gained from wielding, meaning this does 20 when wielded which is as much as security weapons
should probably also have a higher weight class since it can be stored in a janicart

Reply:
Probably meant to make force wielded set to 12 like other inferior servers do
Fixed by making force wielded 4